cli/internal/env/env.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/vercel/turborepo/cli/internal/util"
 )
 
+// Prefixes for common system environment variables that we always exclude from prefix
+var blockedEnvVarList = []string{
+	"_VERCEL_",
```

Comment:
It's not a prefix, it's injected as `NEXT_PUBLIC_VERCEL_*` and `VERCEL_*`

Reply:
Behavior is correct, I'm updating the comment